meters/measurements.go

```diff
@@ -142,99 +145,99 @@ const (
+	ExportPowerL3:    newInternalMeasurement(withDescription("L3 Export Power"), withUnit(units.Watt), withMetricType(Gauge)),
+	ReactivePower:    newInternalMeasurement(withDescription("Reactive Power"), withUnit(units.Var), withMetricType(Gauge)),
+	ReactivePowerL1:  newInternalMeasurement(withDescription("L1 Reactive Power"), withUnit(units.Var), withMetricType(Gauge)),
+	ReactivePowerL2:  newInternalMeasurement(withDescription("L2 Reactive Power"), withUnit(units.Var), withMetricType(Gauge)),
+	ReactivePowerL3:  newInternalMeasurement(withDescription("L3 Reactive Power"), withUnit(units.Var), withMetricType(Gauge)),
+	ApparentPower:    newInternalMeasurement(withDescription("Apparent Power"), withUnit(units.Voltampere), withMetricType(Gauge)),
+	ApparentPowerL1:  newInternalMeasurement(withDescription("L1 Apparent Power"), withUnit(units.Voltampere), withMetricType(Gauge)),
+	ApparentPowerL2:  newInternalMeasurement(withDescription("L2 Apparent Power"), withUnit(units.Voltampere), withMetricType(Gauge)),
+	ApparentPowerL3:  newInternalMeasurement(withDescription("L3 Apparent Power"), withUnit(units.Voltampere), withMetricType(Gauge)),
+	Cosphi:           newInternalMeasurement(withDescription("Power Factor Cosphi"), withMetricType(Gauge)),
+	CosphiL1:         newInternalMeasurement(withDescription("L1 Power Factor Cosphi"), withMetricType(Gauge)),
+	CosphiL2:         newInternalMeasurement(withDescription("L2 Power Factor Cosphi"), withMetricType(Gauge)),
+	CosphiL3:         newInternalMeasurement(withDescription("L3 Power Factor Cosphi"), withMetricType(Gauge)),
+	THD:              newInternalMeasurement(withDescription("Average voltage to neutral THD"), withUnit(units.Percent), withMetricType(Gauge)),
+	THDL1:            newInternalMeasurement(withDescription("L1 Voltage to neutral THD"), withUnit(units.Percent), withMetricType(Gauge)),
+	THDL2:            newInternalMeasurement(withDescription("L2 Voltage to neutral THD"), withUnit(units.Percent), withMetricType(Gauge)),
+	THDL3:            newInternalMeasurement(withDescription("L3 Voltage to neutral THD"), withUnit(units.Percent), withMetricType(Gauge)),
+	Sum:              newInternalMeasurement(withDescription("Total Energy Sum"), withPrometheusName("energy_sum"), withUnit(units.KiloWattHour), withMetricType(Counter)),
+	SumT1:            newInternalMeasurement(withDescription("Tariff 1 Energy Sum"), withUnit(units.KiloWattHour), withMetricType(Counter)),
+	SumT2:            newInternalMeasurement(withDescription("Tariff 2 Energy Sum"), withUnit(units.KiloWattHour), withMetricType(Counter)),
+	SumL1:            newInternalMeasurement(withDescription("L1 Energy Sum"), withUnit(units.KiloWattHour), withMetricType(Counter)),
+	SumL2:            newInternalMeasurement(withDescription("L2 Energy Sum"), withUnit(units.KiloWattHour), withMetricType(Counter)),
+	SumL3:            newInternalMeasurement(withDescription("L3 Energy Sum"), withUnit(units.KiloWattHour), withMetricType(Counter)),
```

Comment:
@andig What does `Total [Energy] Sum` exactly mean? Does it represent a kind of balance between imported/exported energy?

Reply:
I think this is just the `Sum` value that `mbmd` is reading from the meters. Whether this is actually a sum of import and export (probably import - export) is up to the meter manufacturer.